### ISSUE TITLE
As a user, I want to download a binary rather than running from source

### ISSUE THREAD

**Issue by afeld** [opencontrol/fedramp-templater#15]
Installing the dependencies can be painful, so easier to provide a binary that gives everything needed.

/cc https://github.com/opencontrol/fedramp-templater/issues/12

**Comment by mogul** [comment]
I have this feeling I saw this get done...? Or at least the instructions were revised?

**Comment by afeld** [comment]
Nope, just for Masonry: https://github.com/opencontrol/compliance-masonry/pull/232
